config/routes.rb

```diff
@@ -13,6 +13,8 @@
   get "places/:slug", as: 'place', to: 'places#show'
   resources 'places', only: [:create]
 
+  resources 'comments', only: [:create]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.